lang/c++/build.sh

```diff
@@ -58,7 +58,7 @@ function do_doc() {
 function do_dist() {
   rm -rf $BUILD_CPP/
   mkdir -p $BUILD_CPP
-  cp -r api AUTHORS build.sh CMakeLists.txt ChangeLog \
+  cp -r AUTHORS build.sh CMakeLists.txt ChangeLog \
```

Comment:
AFAIR it has been replaced with "include/avro" folder

Reply:
Can you elaborate @martin-g I'm not sure if I'm following you

Reply:
https://github.com/apache/avro/commit/8281e610ab5666bb3ae6e22a2cc2b11af0fb9226

Reply:
I see thanks, I missed that PR 🙏 
```suggestion
  cp -r include/avro AUTHORS build.sh CMakeLists.txt ChangeLog \
```